src/data.js

```diff
@@ -2,9 +2,30 @@ import emojilib from 'emojilib'
 
 import { normalizeCode } from './utils.js'
 
-export const emojiData = Object.entries(emojilib.lib).map(
-  ([name, { char: emoji }]) => [name, emoji]
-)
+export const emojiData = [
+  // First we fill in all possible names for each emoji
+  ...Object.entries(emojilib).flatMap(([emoji, names]) => {
+    return names.map(name => [name, emoji])
+  }),
+  // Then we give priority to the first name for each emoji
+  ...Object.entries(emojilib).flatMap(([emoji, [name]]) => {
+    const results = [[name, emoji]]
+
+    if (name.startsWith('flag_')) {
+      results.push([name.slice('flag_'.length), emoji])
+    }
+
+    if (name.startsWith('smiling_face_with_')) {
+      results.push([name.slice('smiling_face_with_'.length), emoji])
+    }
+
+    if (name.endsWith('_face')) {
+      results.push([name.slice(0, name.length - '_face'.length), emoji])
+    }
+
+    return results
+  }),
+]
```

Comment:
I added a few explicit handlers (e.g. `:mexico:` is no longer an alias of `:flag_mexico:`. Other changes are causing more test failures. I don't know what to do here - do we maintain explicit backwards compatibility? File issues on emojilib? Accept breaking changes? 😬 

cc @omnidan, I don't know what to do 😄

Reply:
Talked on Discord: I'm going to add compatibility helpers for the old aliases like this one. That way this isn't a breaking change for users in the sense of dropping emoji name support.

I'll also look into whether we should file any issues on `emojilib`. https://github.com/muan/emojilib/pull/178 indicated intentionally removing some aliases.

Reply:
(I'm waiting on #112 / https://github.com/muan/emojilib/issues/194)

Reply:
Hi! How is the status of this??

Reply:
WHoooo yeah how is the status...

1. emojilib v3 had a bunch of changes to its keywords - to the point where node-emoji can't directly transition.
2. I pinged the maintainer (muan) and was added as a collaborator there a few weeks ago.
3. What I want to do next is make a comparison of keywords: what exists in unicode, common operating systems, common messaging apps, GitHub, etc. (beyond what muan actually asked for)
4. I haven't found a good tracking table yet that has all that info
    * I sent a request to get access the invite-only Emojipedia API a day or two ago but haven't heard back yet

So there is progress being made, just... slow progress.

Reply:
Hello beautiful humans, I love what you did with this lib and thank you for providing this library for developer like me to build on top of it, is there a way we can get this fix merged? I desprately need some emojis which are missing from your lib. Maybe we can add a `.extend(":emoji-code","✅")` function, just in case some emojis are missing, developer can update it and make it searchable across project?

Reply:
> Maybe we can add a .extend(":emoji-code","✅") function

Oh! I like that idea standalone - could you file a new issue for that please? Let's track it separately so we can maybe get it in much sooner!

Reply:
Filed #144 :)